ssh/tailssh/tailssh_test.go

```diff
@@ -2,7 +2,7 @@
 // Use of this source code is governed by a BSD-style
 // license that can be found in the LICENSE file.
 
-//go:build linux || darwin
+//go:build linux || darwin || freebsd
```

Comment:
???

Reply:
Oh, Tailscale SSH FreeBSD support was added in Tailscale 1.34 so figured it should start running tests too. I'll remove for now and add in its own commit.